tests/test_fio_workload.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import logging
+import copy
+
+from ocs import defaults, exceptions
+from ocs import workload
+from ocsci.testlib import ManageTest
+from utility.utils import TimeoutSampler
+
+
+from tests.fixtures import (
+    create_rbd_storageclass, create_pod, create_pvc, create_ceph_block_pool,
+    create_rbd_secret
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
```

Comment:
You can use fixtures by providing a string value of their name. For example, you can just use 'create_rbd_secret'. here.

Reply:
If I import in above statement and stringify the name here its not working. Hence I had to leave this as is.